draft-ietf-mls-protocol.md

```diff
@@ -3860,171 +3830,3 @@ MLS DE, that MLS DE SHOULD defer to the judgment of the other MLS DEs.
   thyla.van.der@merwe.tech
 
 --- back
-
-
-# Tree Math {#tree-math}
```

Comment:
I might keep this appendix, but change the emphasis:

# Array-Based Tree Representation

The ratchet tree extension defined in {{ratchet-tree-extension}} provides a way to transport a ratchet tree as an array of nodes, such that relationships among the nodes correspond to certain numerical relationships between the indices of the nodes in the array.  These numerical relationship can be used to enable an array-based in-memory representation (in addition to the on-the-wire representation) or to facilitate decoding of the wire-format array into another in-memory representation.  In this appendix, we provide provide Python example code for computing several common relationships among nodes, for example mapping between parents and left/right children.

Reply:
Keeping it with a different emphasis seems fine, however I think it would better be located in a separate "implementation recommendation" document and not in the "protocol" document?

Reply:
I can live with that.  @kkohbrok I think you were going to start an "implementation considerations" document?